build/helm/dc_website_v1/README.md

```diff
@@ -0,0 +1,88 @@
+# Helm Charts
+
+The DC website can be installed into a GKE cluster using [helm](https://helm.sh/).
+
+
+# How to install the DC website on a live GKE cluster.
+
+## Prerequisites
+
+- A live GKE cluster with [workload identiy](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled. This is a one time setup.
+- [A global static IP reserved](https://cloud.google.com/compute/docs/ip-addresses/reserve-static-external-ip-address).
+- Domain(s) from your choice of DNS provider, and have the a [DNS type A record](https://en.wikipedia.org/wiki/List_of_DNS_record_types) which points from the domain to the IP from above.
+
+Note: DNS record takes up to 72 hours to propagate. While the helm chart
+can still be installed before it is completed, the load balancer that is created by the chart may not be ready until then.
+
+
+## Installation
+
+1. Make sure that the k8s credentials are configured. You can visit the GCP UI and click "CONNECT" on your cluster's page to get the command to configure the credentials.
+
+![Alt text](images/cluster.png?raw=true "credentials")
+
+The command below should point to your cluster.
+
+```
+kubectl config current-context
+```
+
+2. Paste the template below into a new file called instance-values.yaml.
+
+```
+website:
+  domains: ["your-website-domain"]
+  githash: "your-website-githash"
+  ingress:
+    annotations:
+      kubernetes.io/ingress.global-static-ip-name: "your-external-global-static-ip-name"
+  flask:
+    secretGCPProjectID: "your-flasks-secret-gcp-project"
+  iap:
+    enabled: true
+    clientID: "your-iap-client-id"
+    secret: "your-iap-secret"
+
+mixer:
+  gcpProjectID: "your-mixer-gcp-project-id"
+  githash: "your-mixer-githash"
+
+namespace:
+  name: "your-namespace"
+
+serviceAccount:
+  annotations: {
+    iam.gke.io/gcp-service-account: "your-robot-google-service-account"
+  }
+
+kgStoreConfig:
+  bigqueryVersion: "your-bigquery-version"
+  bigtableImportGroupsVersion: |
+    <content-of-bigtable-import-group-version-file>
+  storeProjectID: "your-storage-project-gcp-id
+```
+
+3. Fill out all the instance specific variables from above.
```

Comment:
might mention where to find these values and reference the one time setup process?